src/main/java/org/fisco/bcos/channel/dto/BcosBlkNotify.java

```diff
@@ -0,0 +1,36 @@
+package org.fisco.bcos.channel.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigInteger;
+
+public class BcosBlkNotify {
```

Comment:
the class name should be a noun, and all codes belong to bcos, so you don't have to use bcos as a prefix
maybe BlockNotification is more available

Reply:
ok, It has been modified to BcosBlockNotification，Bcos prefix is just use the same style as others .